### ISSUE TITLE
Allow specifying color for errored commands 

### ISSUE THREAD

**Issue by quentin-sommer** [warpdotdev/Warp#2385]
### Describe the solution you'd like?

```text
I would love to have 1 color specifically for this or the ability to turn off the highlight and just keep the lateral red bar
```

### Is your feature request related to a problem? Please describe.

```text
when a command errors the red background highlight can be too strong for my liking, especially when the block is big with lots of output. Sure we can change the red color of the theme but it affects everything.
```

### How important is this feature to you?

3

### Warp Internal (ignore) - linear-label:770f6576-d6c0-4e4f-a259-fc64b5156087

Ignore

**Comment by dannyneira** [comment]
Thanks for this feature request! Please :+1:  and subscribe to this request as it helps us gauge interest. We will post any updates here.

**Comment by dportalesr** [comment]
Indeed! I'd like the ability to complete remove the background color because it reduces the contrast with foreground text. The lateral indicator is more than enough for me.

**Comment by rsalmei** [comment]
Exactly, me too! I hate the red background, to the point it makes me avoid using Warp for now.
Please make it configurable. Maybe only a border I could bear, or that single thick vertical line to the left of the block, but not coloring the whole screen, please... In huge error outputs that is abysmal.

<img width="1012" alt="image" src="https://github.com/warpdotdev/Warp/assets/6652853/b244e71a-3e85-4427-b661-123e6994bd88">

<br><br>
For comparison, the same on iTerm2:
<img width="966" alt="image" src="https://github.com/warpdotdev/Warp/assets/6652853/6df61e17-aea4-44df-8eba-fc5c9a04a2f5">

**Comment by samholmes** [comment]
Also, when you type a command that puts info in stderr (which is the proper unix way), it is red even though the output isn't an "error". For example, most commands will give documentation on how to use the command by default.

**Comment by slackwing** [comment]
Yes this background color is almost never useful to me and is distracting. It makes me feel like my working state is unclean, when no, I intentionally ^C terminated a program. For some reason this almost pushed me over the edge to quit Warp after a couple of months of really trying. I'll push through this but, I hope the Warp devs aren't underestimating the value of customization of appearance (at the very least considering opt-out features by default for _added_, non-neutral appearance features) for retention of terminal users—I feel that the kind of terminal user who cares to try Warp probably cares about this more than the average terminal user. But I don't know, I'm speculating.

**Comment by cricklet** [comment]
I might be weird but FWIW this is a blocker for me (:

I check this issue every few months to see if there's any changes but until then I don't use Warp

**Comment by sauron918** [comment]
It'd be nice to have the ability to specify both selected and errored block colors (and make them transparent as an option)
